README.md

```diff
@@ -89,22 +94,6 @@ return assert.isRejected(promise, Error, "optional message");
 return assert.isRejected(promise, /error message matcher/, "optional message");
 ```
 
-### Progress Callbacks
```

Comment:
that's weird. I meant to delete my explanation of promise assertions, not this section. not sure how that happened. I'll have to fix it.